index.ts

```diff
@@ -606,7 +624,7 @@ commander
             logError(error, ERROR.CREATE);
           });
         });
-      });
+      });  
```

Comment:
Looks like we should also add no-trailing-whitespace (https://palantir.github.io/tslint/rules/no-trailing-whitespace/) to the tslint

Reply:
👍